test/common/pixel-tests

```diff
@@ -1,25 +1,35 @@
 #!/bin/bash
 
-set -eu
-
-SUBDIR=test/reference
-REPO=pixel-test-reference
-
-GITHUB_BASE="${GITHUB_BASE:-cockpit-project/cockpit}"
-GITHUB_REPOSITORY="${GITHUB_BASE%/*}/${REPO}"
-CLONE_REMOTE="https://github.com/${GITHUB_REPOSITORY}"
-PUSH_REMOTE="git@github.com:${GITHUB_REPOSITORY}"
+GITHUB_REPO='pixel-test-reference'
+SUBDIR='test/reference'
 
-message() {
-    [ "${V-}" != 0 ] || printf "  %-8s %s\n" "$1" "$2"
-}
+set -eu
+cd "$(realpath -m "$0"/../../..)"
+. pkg/lib/git-utils.sh
 
 cmd_init() {
     git submodule add -b empty "$CLONE_REMOTE" "$SUBDIR"
 }
 
+cmd_checkout() {
+    local sha="${1-$(get_index_gitlink node_modules)}"
+
+    fetch_sha_to_cache "${sha}"
+    cmd_remove
```

Comment:
I am not sure I like how `pixel-tests pull` might now delete test/reference completely. This will remove untracked files, while `git checkout` and `git submodule update` leave those alone.

On the one hand, removing untracked files will make it impossible for the `pull` to fail because of conflicts, on the other hand, it might throw files away that the developer cares about.

Reply:
See comment above.  I'd like your input to decide what to do here :)